android/quest/src/main/java/org/smartregister/fhircore/quest/ui/family/profile/FamilyProfileViewModel.kt

```diff
@@ -93,6 +95,38 @@ constructor(
     }
   }
 
+  fun filterEligibleFamilyMember(
+    profileViewData: ProfileViewData.FamilyProfileViewData
+  ): ChangeFamilyMembersHolder {
+    val listOfFamilies =
+      profileViewData.familyMemberViewStates.filter {
+        if (it.age.contains("y")) (it.age.split(" ")[0].replace("y", "").toInt() > 15) else false
+      }
```

Comment:
This will not always be the case. The abbreviation can be translated into something different than 'y' when the app language setting is changed. I would suggest you use the `birthDate` to get the date not the formatted age string. Use the extension method provided here against the date of birth to get the years. `org.smartregister.fhircore.engine.util.extension.DateTimeExtensionKt#yearsPassed`